Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -7771,6 +7771,20 @@ UnifyDamagedRegions:
      WebKit:
        default: false
 
+UpdateHistoryByNavigatingFrame:
```

Comment:
Could we use a FrameLoaderClient call instead of adding a preference?  This is just to distinguish WebKitLegacy from modern WebKit, right?  We don't actually need this to be runtime configurable.